oaff/app/oaff/app/request_handlers/collections_list.py

```diff
@@ -12,7 +12,7 @@
 LOGGER: Final = getLogger(__file__)
 
 
-class CollectionsList(RequestHandler):
+class CollectionsList(RequestHandler):  # type: ignore[no-redef]
```

Comment:
In this and the following instances I think it would be preferable to rename the types for clarity. e.g. 
`from oaff.app.requests.collections_list import CollectionsList`
becomes
`from oaff.app.requests.collections_list import CollectionsList as CollectionsListRequest`
and 
`class CollectionsList(RequestHandler)`
becomes
`class CollectionsListRequestHandler(RequestHandler)`

This was an oversight on my part and I think more explicit naming provides a significant advantage here